### PR TITLE
Initial the Azure.ResourceManager.NetworkFunction

### LINKED ISSUE #29240: Initial Azure.ResourceManager.NetworkFunction

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

**Review by Pablo1q — APPROVED** [review]

File: sdk/networkfunction/Azure.ResourceManager.NetworkFunction/src/Generated/Extensions/NetworkFunctionExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+        /// Request Path: /providers/Microsoft.NetworkFunction/operations
+        /// Operation Id: NetworkFunction_ListOperations
+        /// </summary>
+        /// <param name="tenantResource"> The <see cref="TenantResource" /> instance the method will execute against. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <returns> An async collection of <see cref="CollectorOperation" /> that may take multiple service requests to iterate over. </returns>
+        public static AsyncPageable<CollectorOperation> GetOperationsNetworkFunctionsAsync(this TenantResource tenantResource, CancellationToken cancellationToken = default)
+        {
+            return GetExtensionClient(tenantResource).GetOperationsNetworkFunctionsAsync(cancellationToken);
+        }
```
Comment (fengzhou-msft) [inline sdk/networkfunction/Azure.ResourceManager.NetworkFunction/src/Generated/Extensions/NetworkFunctionExtensions.cs:42]: This operation should be removed.

File: sdk/networkfunction/Azure.ResourceManager.NetworkFunction/api/Azure.ResourceManager.NetworkFunction.netstandard2.0.cs [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+    public partial class IngestionPolicyPropertiesFormat
+    {
+        public IngestionPolicyPropertiesFormat() { }
+        public System.Collections.Generic.IList<Azure.ResourceManager.NetworkFunction.Models.IngestionSourcesPropertiesFormat> IngestionSources { get { throw null; } }
+        public Azure.ResourceManager.NetworkFunction.Models.IngestionType? IngestionType { get { throw null; } set { } }
+    }
+    public partial class IngestionSourcesPropertiesFormat
+    {
+        public IngestionSourcesPropertiesFormat() { }
+        public string ResourceId { get { throw null; } set { } }
```
Comment (fengzhou-msft) [inline sdk/networkfunction/Azure.ResourceManager.NetworkFunction/api/Azure.ResourceManager.NetworkFunction.netstandard2.0.cs:196]: Is it a ResourceIdentifier?

Comment (ArthurMa1978) [reply sdk/networkfunction/Azure.ResourceManager.NetworkFunction/api/Azure.ResourceManager.NetworkFunction.netstandard2.0.cs:196]: It's `ResourceIdentifier` when `SourceType` is Resource, but other values for other types.

File: sdk/networkfunction/Azure.ResourceManager.NetworkFunction/src/Generated/Extensions/NetworkFunctionExtensions.cs (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/networkfunction/Azure.ResourceManager.NetworkFunction/src/Generated/Extensions/NetworkFunctionExtensions.cs]: fixed

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**